### ISSUE TITLE
Failure to link ok

### ISSUE THREAD

**Issue by ybh0822** [berkeley-dsep-infra/datahub#1740]
Hello, I am a GSI for Physics188/288 (https://classes.berkeley.edu/content/2020-fall-physics-188-001-lec-001).
We have been using ok for homework submission, but this year, a number of students couldn't link to ok with the following error message. We couldn't figure out how to resolve this issue.
![okpy issue](https://user-images.githubusercontent.com/14793188/91595892-3d8dda80-e99f-11ea-8115-b57a0c7bbff9.png)
My apologies if I should have contacted a different personnel. Thank you :)

**Comment by davidwagner** [comment]
We saw something similar Data 8.  When we pushed an update to a homework after some students had downloaded it, for reasons I don't understand some of them saw the same error.  Might be related to https://github.com/okpy/ok-client/issues/348.  A workaround is to have students manually replace "_ = ok.auth(inline=True)" with "_ = ok.auth(inline=True, force=True)" if they experience this issue and re-run the cell.

Another cause of problems for some students was that when they initiated the okpy authentication and pasted their auth token into the input box, they pressed "Run" instead of pressing "enter", and that caused problems.  Asking them to restart their kernel and then do it again but this time pressing enter fixed it for them.
